mkdocs/docs/HPC/getting_started.md

```diff
@@ -0,0 +1,209 @@
+    
+    replace <b>{{userid}}</b> with your id (see <https://account.vscentrum.be>)
+
+!!! Info
+
+    When having problems see the [connection issues](troubleshooting.md#sec:connecting-issues) section on the [troubleshooting page](troubleshooting.md).
+
+!!! Info
+
+    You can also still use the ugent web portal (see [shell access on web portal](web_portal.md#shell-access))
+    
+
+{%- endif %}
+
+### Transfer your files
+
+Now that you can login, it's time to transfer files from your local pc to your **Home Directory** on a cluster.
+
+Get [TensorFlow-2.x_mnist-test.py](https://raw.githubusercontent.com/EESSI/eessi-demo/main/TensorFlow/TensorFlow-2.x_mnist-test.py) 
+and [run.sh](https://raw.githubusercontent.com/EESSI/eessi-demo/main/TensorFlow/run.sh) from the [example](https://github.com/EESSI/eessi-demo/tree/main/TensorFlow) on your local machine.
+
+{%- if OS == windows %}
+
+The [HPC-Ugent web portal](https://login.hpc.ugent.be) provides a filebrowser that allows uploading files.
+For more information see the [file browser section](web_portal.md#file-browser).
+
+Upload the two files to your **Home Directory** and go back to your shell.
+
+!!! Info
+
+    As an alternative, you can use WinSCP (see [our section](connecting.md#winscp))
+
+{%- else %}
+
+On your local machine you can run:
+<pre><code>wget https://raw.githubusercontent.com/EESSI/eessi-demo/main/TensorFlow/TensorFlow-2.x_mnist-test.py
+wget https://raw.githubusercontent.com/EESSI/eessi-demo/main/TensorFlow/run.sh
+</code></pre>
+
+Now using `scp` the files can be copied from your local host to your *home directory* (~) on the remote host (HPC).
+<pre><code>scp TensorFlow-2.x_mnist-test.py run.sh {{userid}}{{ loginnode }}:~ </code></pre>
+<pre><code>ssh  {{userid}}@{{ loginnode }} </code></pre>
+
+!!! Info
+
+    For more information about transfering files or `scp`, see [tranfer files from/to hpc](connecting.md#transfer-files-tofrom-the-hpc).
+
+{%- endif %}
+
+When running `ls` on your HPC shell, you should see the two files listed:
+
+```shell
+$ ls
+run.sh TensorFlow-2.x_mnist-test.py
+```
+
+!!! Warning
+
+    When you do not see these files, make sure you uploaded the files to your **Home Directory**.
+
+### Submitting a job
+
+Jobs are submitted and executed using job scripts. In our case **run.sh**.
+
+A job script is a shell script, a text file that specifies the resources, software that is used (via module load) and the steps that should be done for the calculation.
+
+Our job script looks like this:
+
+<center>-- run.sh --</center>
+
+```bash
+#!/bin/bash
+
+module load TensorFlow/2.3.1-foss-2020a-Python-3.8.2
+
+time python TensorFlow-2.x_mnist-test.py
+
+```
+<sub>As you can see this job script will run the python program: **TensorFlow-2.x_mnist-test.py**</sub>
+
+
+The jobs you submit are per default executed on **cluser/{{defaultcluster}}**, you can swap cluster by the following command.
+
+<pre><code>module swap cluster/{{othercluster}}</code></pre>
```

Comment:
This is a good place to link to the chapter on the debug/interactive cluster, and recommending to use that (since there's no waiting time there when using a limited amount of resources).